src/test/java/org/folio/readingroom/service/ServicePointServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package org.folio.readingroom.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.UUID;
+import org.folio.readingroom.client.feign.ServicePointClient;
+import org.folio.readingroom.service.impl.ServicePointServiceImpl;
+import org.folio.spring.model.ResultList;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class ServicePointServiceTest {
+  @Mock
+  ServicePointClient servicePointClient;
+  @InjectMocks
+  ServicePointServiceImpl servicePointServiceImpl;
+
+  List<UUID> servicePointIds;
+  List<ServicePointClient.InventoryServicePoint> inventoryServicePoints;
+  ResultList<ServicePointClient.InventoryServicePoint> resultList;
+
+  @BeforeEach
+  void initData() {
+    servicePointIds = Arrays.asList(UUID.randomUUID(), UUID.randomUUID());
+  }
+
+  @Test
+  void fetchInvalidServicePointList_WhenAllPointsAreValid() {
```

Comment:
Please change the name from fetchInvalidServicePointList_WhenAllPointsAreValid to fetchInvalidServicePointList_WhenAllServicePointsAreValid.
Please apply the same in all methods